src/panels/lovelace/editor/config-elements/hui-entities-card-editor.ts

```diff
@@ -105,28 +87,15 @@ export class HuiEntitiesCardEditor extends LitElement
       return html``;
     }
 
-    if (this._editRowConfig) {
+    if (this._subElementEditorConfig) {
       return html`
-        <hui-detail-editor-base
+        <hui-sub-element-editor
           .hass=${this.hass}
-          .guiModeAvailable=${this._editRowGuiModeAvailable}
-          .guiMode=${this._editRowGuiMode}
-          @toggle-gui-mode=${this._toggleMode}
+          .config=${this._subElementEditorConfig}
           @go-back=${this._goBack}
+          @config-changed=${this._valueChanged}
```

Comment:
Let's not use `_valueChanged` here, that function is getting really complicated and unreadable now. Create a new function for the sub config handling.